doc/ref/fpsemi.xml

```diff
@@ -20,9 +21,9 @@ by a set of relations (a generating set for the congruence), i.e., a set of
 pairs of words in the free semigroup.
 <P/>
 <Example><![CDATA[
-gap> f:=FreeSemigroup("a","b");;
-gap> x:=GeneratorsOfSemigroup(f);;
-gap> s:=f/[ [x[1]*x[2],x[2]*x[1]] ];
+gap> f := FreeSemigroup( "a", "b" );;
+gap> x := GeneratorsOfSemigroup(f);;
+gap> s := f/[ [x[1]*x[2], x[2]*x[1]] ];
```

Comment:
This should either be `f / [ [ x[1] * x[2], x[2] * x[1] ] ];` or `f / [[x[1] * x[2], x[2] * x[1]]];` but not a combination of both.